nextcloudappstore/core/templates/nav.html

```diff
@@ -41,6 +41,11 @@
                 <li class="nav-link">
                     <a href="{% url 'home' %}">{% trans 'Apps' %}</a>
                 </li>
+                {% if request.user.is_authenticated %}
```

Comment:
ok :)

Would you be so kind to grep for is_authenticated() :D? we still go that all over our template (see https://docs.djangoproject.com/en/1.10/releases/1.10/#user-is-auth-anon-deprecation)